code/modules/urist/modules/jungle/planet_turfs.dm

```diff
@@ -744,3 +751,117 @@
 	color = "#b75d0e"
 	desc = "Arid, sandy Martian dirt."
 	temperature = 288.15 //15C
+
+//temperate forest turfs
+
+/turf/simulated/floor/planet/temperate
+	temperature = 283.15 //10C
```

Comment:
Fine as-is, but to avoid having to do the maths and comment in the future, you can use the `T0C` macro, which is short for 0 degrees C.
In this case `temperature = T0C+10`